doc/source/documentation_style/docstrings.rst

```diff
@@ -24,14 +29,45 @@ PyAnsys library docstrings contain at a minimum the following
 * `Returns <https://numpydoc.readthedocs.io/en/latest/format.html#returns>`_ if applicable
 * `Examples <https://numpydoc.readthedocs.io/en/latest/format.html#examples>`_
 
-These sections should follow the numpydoc standards. To avoid
+These sections should follow numpydoc standards. To avoid
 inconsistencies between PyAnsys libraries, adhere to the additional 
 standards that follow.
 
+Classes
+~~~~~~~
+A class is a "noun" representing a collection of methods. For consistency within PyAnsys libraries,
+always start the short description for a class with a verb ending in "s", followed by an extended
+summary if applicable::
+
+  class FieldAnalysis3D(Analysis, object):
+    """Manages 3D field analysis setup in HFSS, Maxwell 3D, and Q3D.
+
+    This class is automatically initialized by an application call from one of
+    the 3D tools. See the application function for parameter definitions.
+
+
+Always have a line break between the end of a class docstring and the subsequent methods.
+
+Methods
+~~~~~~~
+A method is a "verb" representing an action that can be performed. For consistency within PyAnsys
+libraries, always start the short description for a method with a verb not ending in "s", followed
+by an extended summary if applicable::
+
+  def export_mesh_stats(self, setup_name, variation_string="", mesh_path=None):
+        """Export mesh statistics to a file.
+
+
+If a method has the decorator ``@exceperty``, you can remove the Parameters section for this method.
```

Comment:
I'm not sure what this means.. maybe it should be ``@property``?

Reply:
@greschd maybe? I'm working from notes I took over six months ago. I copied what I had but it does seem like maybe it should have said property. LOL!

Reply:
For ``@property`` I think this advice would make sense, because it turns a method into a "computed variable". Those can't have parameters.